app/src/editor/shared/components/modules/Map.jsx

```diff
@@ -238,9 +234,23 @@ export default class MapModule extends React.Component<Props, State> {
         const { queuedMarkers } = this
         this.queuedMarkers = {}
 
-        this.setState((state) => ({
-            markers: merge(state.markers, queuedMarkers),
-        }))
+        this.setState((state) => {
+            const markers = { ...state.markers }
+            // $FlowFixMe Object.values() returns mixed[]
+            Object.values(queuedMarkers).forEach((m: Marker) => {
```

Comment:
Well, you could avoid this `$FlowFixMe` by doing something like

```js
Object.values(queuedMarkers).forEach((mkr: any) => {
    const m: Marker = mkr
    // …
})
```

Again, 2 lines tho, 2 vars for the same. Long story short, ugly!